pkgs/development/python-modules/flaxlib/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  flax,
+  rustPlatform,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "flaxlib";
+  version = "0.0.1-a1";
+  pyproject = true;
+
+  inherit (flax) src;
+
+  sourceRoot = "${src.name}/flaxlib";
+
+  postPatch = ''
+    expected_version="${version}"
+    actual_version=$(grep -oP '(?<=version = ")[^"]*' Cargo.toml)
```

Comment:
I think using tomlq (part of yq) here would be more robust

Reply:
Good idea, done.